.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+
+
+<!--
+Thank you for your Pull Request!
+
+If you are new to contributing to Mono, please try to do your best at conforming to our coding guidelines http://www.mono-project.com/community/contributing/coding-guidelines/ but don't worry if you get something wrong. One of the project members will help you to get things landed.
+
+Does your pull request fixes any of existing issues? Please use Fixes #issue-number format.
```

Comment:
typo: Does your pull request ~fixes~ fix any of *the* existing issues?

Maybe also rephrase the second sentence to `Please use the following format: Fixes #issue-number`